### ISSUE TITLE
RealExpression display does not use all the available space

### ISSUE THREAD

**Issue by ceraolo** [OpenModelica/OpenModelica#11980]
### Description
Consider the following screenshot:
![image](https://github.com/OpenModelica/OpenModelica/assets/7373030/7a7ccf2c-061a-469c-9a51-332844313376)

the upper RealExpression does not use all the available space to show the string.

### Expected Behavior
Real expression should use all the available space to display the string.

### OM version
OM 1.23.0-dev310

**Comment by adeas31** [comment]
Please share the test model.

**Comment by ceraolo** [comment]
here you are:
```
model m
  Modelica.Blocks.Sources.RealExpression uf11(y = gen11x2.ufPuInStart) annotation(
    Placement(transformation(origin = {-9, 51}, extent = {{-15, -7}, {15, 7}})));
  Modelica.Blocks.Sources.RealExpression uf111(y = gen11x2.ufPuInStart)  annotation(
    Placement(transformation(origin = {-9, 31}, extent = {{-15, -13}, {15, 13}})));
equation

annotation(
    uses(Modelica(version = "4.0.0")));
end m;
```

**Comment by adeas31** [comment]
@MaksKBV and @AustraleanPerson any idea? I think the calculated scale in #11691 is too small.

**Comment by AustraleanPerson** [comment]
@MaksKBV will fix it after 25/03.

**Comment by AustraleanPerson** [comment]
I tried replacing
`QRectF(xtl * curScale, ytl * curScale, boundingRectangle.width() * curScale, boundingRectangle.height() * curScale);`
with
`QRectF(xtl * xScale, ytl * yScale, boundingRectangle.width() * xScale, boundingRectangle.height() * yScale);`
It helps and seems logical.
But i dont know if it will break something.

**Comment by adeas31** [comment]
It seems more correct. I will test a bit and make a PR of it.

**Comment by adeas31** [comment]
Fixed with #12093 and ported to maintenance 1.22 #12095
